python/lsst/ctrl/bps/htcondor/lssthtc.py

```diff
@@ -1422,7 +1422,11 @@ def read_node_status(wms_path):
                     job["DAGNodeName"] = jclassad["Node"]
                     job["bps_job_label"] = label
 
-                    jobs[str(job_id)] = job
+                    jobs[job_id] = job
+                    try:
+                        jobs[job_id] |= loginfo[job_id]
```

Comment:
With this change, this function is also returning information from the dag nodes log file as well as the node status file.  Please update the description in the docstring for the return value to note this.